src/themes/global-style.ts

```diff
@@ -35,6 +35,12 @@ const GlobalStyle = createGlobalStyle`
         padding: 0;
         background: var(--color-black);
     }
+    *[dir='rtl'] {
```

Comment:
```suggestion
    *[langdirection='rtl'] {
```
Can we rename it to langdir or langdirection? It will be much clearer on what it does

Reply:
unfortunately we can't since we are setting the `dir` html attribute with `helmet`, but I'll provide a comment on top it and explain what it does.